test/util.go

```diff
@@ -133,9 +134,7 @@ func alterFiles(dir string) error {
 		case r == 0 && comps > 2:
 			// Delete every tenth file or directory, except top levels
 			err := removeAll(path)
-			if err != nil {
-				return err
-			}
+			return err
```

Comment:
These could all be simplified to just `return removeAll(path)` and so on, but we can leave that for a future cleanup. :)